app/assets/stylesheets/application.scss

```diff
@@ -17,5 +17,22 @@
  */
 .cancel {
   text-align: center;
+}
+
+body {
+ background-image: image_url("");
+ background-size: cover;
```

Comment:
Line should be indented 2 spaces, but was indented 1 spaces<br>Properties should be ordered background-image, background-position, background-position, background-repeat, background-size